lib/modules/datasource/jsr/schema.ts

```diff
@@ -8,13 +9,24 @@ export const JsrPackageMetadata = z
     versions: z.record(
       z.string(),
       z.object({
+        createdAt: MaybeTimestamp,
         yanked: z.boolean().optional(),
       }),
     ),
   })
   .transform(({ versions, latest }): Release[] => {
     return Object.entries(versions).map(([version, val]) => ({
       version,
+      ...{
+        releaseTimestamp:
+          // In the JSR.io API, if a package has had any version published on or after 2025-09-18,
+          // the createdAt field will be returned for all versions of that package, including historical ones.
+          // If a package has not been published at all since this date,
+          // the createdAt field will be omitted entirely from the API response.
+          // Therefore, we can assume that it published no less than that date for older versions
+          // https://github.com/jsr-io/jsr/pull/1194#issuecomment-3522729482
```

Comment:
This is very interesting! I think we should add a `releaseTimestampNote` ([i.e.](https://github.com/renovatebot/renovate/blob/0188496bf12671a5d7503501c51d8cb2195bc9cf/lib/modules/datasource/dotnet-version/index.ts#L27-L28)) in `lib/modules/datasource/jsr/index.ts`, as this will then populate in the documentation ([i.e.](https://docs.renovatebot.com/modules/datasource/dotnet-version/)'s table at the top)